Vagrantfile

```diff
@@ -26,10 +28,9 @@ Vagrant.configure("2") do |config|
     sudo apt-get install -y opam m4 libxen-dev git
     opam init -a --compiler=4.02.3 -y
     opam remote add xs-opam git://github.com/xapi-project/xs-opam
-    opam install -y merlin ocp-indent utop lwt_react depext camlp4
+    opam install -y merlin ocp-indent ocp-index ocp-browser utop lwt_react depext camlp4
     sudo opam depext -y xapi
-    opam pin add xenctrl 0.9.32
-    opam install ezxenstore
```

Comment:
Because ezxenstore is now a xapi dependency, so it doesn't need installing manually.